test/runtest.py

```diff
@@ -779,9 +779,6 @@ def testGetDict(self):
         expected = {str(i): i for i in range(10)}
         self.assertEqual(result, expected)
 
-    @unittest.skipIf(
-        os.environ.get("RAY_USE_XRAY") == "1",
-        "This test does not work with xray yet.")
     def testWait(self):
```

Comment:
What happened to the other test cases that should now pass? I forget exactly why ones, but I think there was one like `testMultipleWaitsAndGets`?

Reply:
I've enabled all wait-related tests.